Filters/ProductSizeFilterForm.cs

```diff
@@ -0,0 +1,52 @@
+﻿
+using System;
+using System.Web.Mvc;
+using Orchard.DisplayManagement;
+using Orchard.Forms.Services;
+using Orchard.Localization;
+using Orchard.Projections.FilterEditors.Forms;
+
+namespace Nwazet.Commerce.Filters {
+
+    public class ProductSizeFilterForm : IFormProvider {
+        public Localizer T { get; set; }
+        protected dynamic Shape { get; set; }
+
+        public ProductSizeFilterForm(IShapeFactory shapeFactory) {
+            Shape = shapeFactory;
+            T = NullLocalizer.Instance;
+        }
+        public void Describe(DescribeContext context) {
+            Func<IShapeFactory, dynamic> form =
+                shape => {
+                    var f = Shape.Form(
+                        Id: "ProductSizeFilterForm",
+                        _ProductFilterForm: Shape.TextBox(
+                            Id: "Value", Name: "Value",
+                            Title: T("The Product Size"),
+                            Description: T("Product Size."),
+                            Classes: new[] { "text", "tokenized" }
+                        ),
+                        _Operator: Shape.SelectList(
+                            Id: "operator", Name: "Operator",
+                            Title: T("Operator"),
+                            Size: 1,
+                            Multiple: false
+                        ));
+                    f._Operator.Add(new SelectListItem { Value = Convert.ToString(StringOperator.Equals), Text = T("Is equal to").Text });
+                    f._Operator.Add(new SelectListItem { Value = Convert.ToString(StringOperator.NotEquals), Text = T("Is not equal to").Text });
+                    f._Operator.Add(new SelectListItem { Value = Convert.ToString(StringOperator.Contains), Text = T("Contains").Text });
```

Comment:
Those contains operators sound very expensive. We might want to warn about the potential negative impact on the query's perf.